modules/icgc-argo-workflows/checkinput/main.nf

```diff
@@ -37,6 +37,11 @@ process CHECKINPUT {
             $samplesheet \\
             samplesheet.valid.csv
         ;;
+    'RNA Alignment')
+        rnaaln.py \\
+            $samplesheet \\
+            samplesheet.valid.csv
+        ;;
     'Germline Variant Call')
         germlinevar.py \\
             $samplesheet \\
```

Comment:
@guanqiaofeng sorry I just realized that these updates need to go to module `checkinput` instead of updating directly in the subworkflow `stage_input`
Once the module got updated, the subworkflow will import the updated module.